frappe/patches/v11_0/set_primary_key_in_series.py

```diff
@@ -0,0 +1,27 @@
+import frappe
+
+def execute():
+    keys_encountered = set()
+    #if current = 0, simply delete the key as it'll be recreated on first entry
+    frappe.db.sql('delete from `tabSeries` where current = 0')
+    duplicate_keys = frappe.db.sql('''
+        SELECT distinct name, current
+        from
+            `tabSeries`
+        where
+            name in (Select name from `tabSeries` group by name having count(name) > 1)
+    ''', as_dict=True)
+    for row in duplicate_keys:
```

Comment:
maybe just keep the one with the higher counter?

Reply:
have added commits for this